src/main/java/ChatBBT.java

```diff
@@ -0,0 +1,88 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.channels.ScatteringByteChannel;
+import java.util.Scanner;
+import todolist.ToDoList;
+
+public class ChatBBT {
+
+    /**
+     * Print the exit message
+     */
+    public static void exitMessage() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Print the message divider
+     */
+    public static void messageDivider() {
+        System.out.println("------------------------------------------");
+    }
+
+    /**
+     * Print the greeting message
+     */
+    public static void greetingMessage() {
+        System.out.println("Hello! I'm ChatBBT");
+        System.out.println("What can I do for you?");
+        messageDivider();
+    }
+
+    /**
+     * Print the feature intro message
+     */
+    public static void featureIntroMessage() {
+        System.out.println("[list] Enter Todo List feature");
+        System.out.println("[bye] Quit");
+        System.out.println("Please enter a command:");
+        messageDivider();
+    }
+
+    public static void createLocalDataFile() {
+        String filePath = "ChatBBTData.txt";
+        try {
+            // Create a File object
+            File file = new File(filePath);
+            // Check if the file already exists
+            if (file.createNewFile()) {
+                System.out.println("File created: " + file.getAbsolutePath());
+                FileWriter fw = new FileWriter("ChatBBTData.txt");
```

Comment:
Per the comment above, this could reference that constant value so that magic strings can be avoided